cmake/config.hh.in

```diff
@@ -1,15 +1,37 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 /* Config.hh. Generated by CMake for @PROJECT_NAME@. */
 
+#ifndef GZ_TOOLS_CONFIG_HH_
+#define GZ_TOOLS_CONFIG_HH_
+
 /* Version number */
-#define ${PROJECT_NAME_UPPER}_MAJOR_VERSION ${PROJECT_MAJOR_VERSION}
-#define ${PROJECT_NAME_UPPER}_MINOR_VERSION ${PROJECT_MINOR_VERSION}
-#define ${PROJECT_NAME_UPPER}_PATCH_VERSION ${PROJECT_PATCH_VERSION}
```

Comment:
Should we keep the `IGNITION` version for ticking before tocking?

Reply:
Actually I don't really see these being used anywhere..

Reply:
Oh I was treating them as part of the public API, but actually it looks like this file isn't being generated and can be removed altogether.

Reply:
* https://github.com/gazebosim/gz-tools/pull/91